core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/domain/common/accessor/MemberAccessorFactory.java

```diff
@@ -34,24 +34,46 @@
     static final String CLASSLOADER_NUDGE_MESSAGE = "Maybe add getClass().getClassLoader() as a parameter to the " +
             SolverFactory.class.getSimpleName() + ".create...() method call.";
 
+    /**
+     * As defined by {@link #buildMemberAccessor(Member, MemberAccessorType, Class, DomainAccessType, Map)},
+     * but will cache the result in the map if provided.
```

Comment:
```suggestion
     * but caches the result in the map if provided.
```

Docs style guidelines: use active tense (avoid past and future tense)